From 875e4137949a17e499db604def49b2cbfae72d53 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.jz@renesas.com>
Date: Mon, 30 Nov 2020 15:05:58 +0700
Subject: [PATCH 005/135] dt-binding: arm: cpus: Document for Arm CortexA55

Document for Arm CortexA55 supported as one of Armv8 Family CPU

Signed-off-by: Binh Nguyen <binh.nguyen.jz@renesas.com>
---
 Documentation/devicetree/bindings/arm/cpus.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/cpus.txt b/Documentation/devicetree/bindings/arm/cpus.txt
index 96dfccc..613a958 100644
--- a/Documentation/devicetree/bindings/arm/cpus.txt
+++ b/Documentation/devicetree/bindings/arm/cpus.txt
@@ -157,6 +157,7 @@ described below.
 			    "arm,cortex-a15"
 			    "arm,cortex-a17"
 			    "arm,cortex-a53"
+			    "arm,cortex-a55"
 			    "arm,cortex-a57"
 			    "arm,cortex-a72"
 			    "arm,cortex-a73"
-- 
2.7.4

